lib/engine/game/base.rb

```diff
@@ -248,6 +248,7 @@ class Base
       DISCARDED_TRAINS = :discard # discard or remove
       DISCARDED_TRAIN_DISCOUNT = 0 # percent
       CLOSED_CORP_TRAINS_REMOVED = true
+      CLOSED_CORP_TOKENS_REMOVED = true
```

Comment:
Isn't this just a dup of the line above?

Reply:
Yeah, bad merge.